### PR TITLE
Update intial index page with admin panel design

### PR DESCRIPTION
**Pull request by Arzhem**

### CODE REVIEW

File: Front-End/index.html [hunk 1]
```diff
@@ -2,18 +2,184 @@
...
+                    <td>22°C</td>
+                    <td style="color: #cd0012;">Inactive</td>
+                </tr>
+                <!--/items-->
+                </tbody>
+
+            </table>
+        </div>
+    </div>
+    <div class="footer">
```
Comment (DimitarSamarov07) [inline Front-End/index.html]: The .footer and .copyright should be a under a footer tag. Read about semantic HTML.

File: Front-End/index.html [hunk 2]
```diff
@@ -2,18 +2,184 @@
...
 </head>
 <body>
-<h1>CONTROL PANEL</h1>
-<p>The panel for our site.</p>
-<img src="https://prismplus.my/cdn/shop/files/AirconLuna-Web-Thumb3_530x@2x.jpg?v=1714356237" alt="aircon" height="200"
-     class="left-aligned">
-<p class = "red">COMING SOON</p>
-<button class="butonn" type="button">SUBMIT</button>
-<button class="butonn-destroy" type="button">Destroy everything</button>
+    <nav class="navbar">
```
Comment (DimitarSamarov07) [inline Front-End/index.html]: Have a clear separation of semantic HTML tags. First, you have the header, then the body and finally the footer. Add the semantic tags and separate accordingly.

File: Front-End/public/styles.css [hunk 3]
```diff
@@ -1,29 +1,204 @@
...
+.brand .logo {
+    display: flex;
+}
+
+.brand .logo img {
+    margin: 5px;
+    width: 2.7rem;
+}
+
+.footer {
```
Comment (DimitarSamarov07) [inline Front-End/public/styles.css]: The footer should have a fixed position - it should always be at the bottom of the page. The way you've coded it, it's dependant on the body size. Fix that. Either extend the body to the height of the page so the footer stays at the bottom or somehow use absolute positioning. The footer has to not look out of place. If you are not able to replicate the issue on your end, zoom out the web page.

File: Front-End/index.html [hunk 4]
```diff
@@ -2,18 +2,184 @@
...
+                <li><a href="#">Google Play</a></li>
+                <li><a href="#">Apple Store</a></li>
+            </ul>
+            <ul>
+                <li><p class="list-head"><a href="https://github.com/DimitarSamarov07/esp-cloud-remote" target="_blank">View us on GitHub</a></p></li>
+            </ul>
 
+        </div>
+    </div>
+    <div class="copyright">
```
Comment (DimitarSamarov07) [inline Front-End/index.html]: Should be a part of the footer.

File: Front-End/public/styles.css [hunk 5]
```diff
@@ -1,29 +1,204 @@
...
+    background-color: #ffffff;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5% 5% 0.5% 5%;
+}
+
+.navbar .logo a {
+    color: #000;
+    text-decoration: none;
```
Comment (DimitarSamarov07) [inline Front-End/public/styles.css]: You use this quite a lot. Why don't you make it a global rule for the anchor tags?

File: Front-End/public/styles.css [hunk 6]
```diff
@@ -1,29 +1,204 @@
...
+}
+
+
+/*navigation bar*/
+.navbar {
+    background-color: #ffffff;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5% 5% 0.5% 5%;
```
Comment (DimitarSamarov07) [inline Front-End/public/styles.css]: Using percentages for padding in this case isn't the best idea as it's not clear what this means(a percentage of what and why?). Either use pixels or something else.

File: Front-End/public/styles.css [hunk 7]
```diff
@@ -1,29 +1,204 @@
...
+.footer-container td {
+    padding: 0 1rem;
+}
+
+.footer .links {
+    display: flex;
+}
+
+.footer .links ul {
+    list-style: none;
```
Comment (DimitarSamarov07) [inline Front-End/public/styles.css]: You use list-style quite a few times. Make it a global rule for ul, so you don't have to repeat it.

File: Front-End/index.html [hunk 8]
```diff
@@ -2,18 +2,184 @@
...
+                        <th>Temperature</th>
+                        <th>Status</th>
+                    </tr>
+                </thead>
+                <tbody>
+                <!--items-->
+                <tr>
+                    <td class="esp-profile">
+                        <img src="public/images/esp-list-logo.png">
+                        <span>ESP-8266</span>
```
Comment (DimitarSamarov07) [inline Front-End/index.html]: Kindly avoid using spans. Please replace them with something else.

File: Front-End/index.html [hunk 9]
```diff
@@ -2,18 +2,184 @@
...
+            </ul>
+            <ul>
+                <li><p class="list-head"><a href="https://github.com/DimitarSamarov07/esp-cloud-remote" target="_blank">View us on GitHub</a></p></li>
+            </ul>
 
+        </div>
+    </div>
+    <div class="copyright">
+        <hr>
+        <span>ESP Cloud Remote @ 2024</span>
```
Comment (DimitarSamarov07) [inline Front-End/index.html]: Change the span, please.

File: Front-End/index.html [hunk 10]
```diff
@@ -2,18 +2,184 @@
...
+                <!--items-->
+                <tr>
+                    <td class="esp-profile">
+                        <img src="public/images/esp-list-logo.png">
+                        <span>ESP-8266</span>
+                    </td>
+                    <td>Cold</td>
+                    <td>2x</td>
+                    <td>22°C</td>
+                    <td style="color: #00c774;">Active</td>
```
Comment (DimitarSamarov07) [inline Front-End/index.html]: Never use inline style tags! Keep the CSS in the CSS file.

**Review by DimitarSamarov07 — CHANGES_REQUESTED** [review]
Let me know if you have any questions.

**Review by Arzhem — COMMENTED** [review]
fixed

**Review by DimitarSamarov07 — APPROVED** [review]
Looks good!

**Pull request merged**